bands/models.py

```diff
@@ -18,81 +18,111 @@ class Event:
 class Band(MagModel):
     group_id = Column(UUID, ForeignKey('group.id'))
     event_id = Column(UUID, ForeignKey('event.id'), nullable=True)
+    info = relationship('BandInfo', backref='band', uselist=False)
+    bio = relationship('BandBio', backref='band', uselist=False)
+    taxes = relationship('BandTaxes', backref='band', uselist=False)
+    stage_plot = relationship('BandStagePlot', backref='band', uselist=False)
+    panel = relationship('BandPanel', backref='band', uselist=False)
+    merch = relationship('BandMerch', backref='band', uselist=False)
+    charity = relationship('BandCharity', backref='band', uselist=False)
 
     payment = Column(Integer, default=0, admin_only=True)
-    vehicles = Column(Integer, default=1, admin_only=True)
-    estimated_loadin_minutes = Column(Integer, default=60, admin_only=True)
-    estimated_performance_minutes = Column(Integer, default=60, admin_only=True)
 
+    @property
+    def all_badges_claimed(self):
+        return not any(a.is_unassigned for a in self.group.attendees)
+
+    @property
```

Comment:
This actually works now that I'm passing in `band_info` as a session object instead of just a DB object, but upon further reflection, I like the admin-based properties to stay in the Band class.